Sources/SwifterSwift/SwiftStdlib/ArrayExtensions.swift

```diff
@@ -6,6 +6,40 @@
 //  Copyright © 2016 SwifterSwift
 //
 
+#if canImport(CoreLocation)
+import CoreLocation
+#endif
+
+// MARK: - Enums
+public extension Array where Element: CLLocation {
+
+    /// SwifterSwift: Distance unit.
+    ///
+    /// - meter: Unit of length - meter.
+    /// - kilometer: Unit of length - kilometer.
+    /// - feet: Unit of length - feet.
+    /// - yard: Unit of length - yard.
+    /// - mile: Unit of length - mile.
+    enum DistanceUnit {
```

Comment:
I think we should avoid create new types here.
Also, we can use the foundation `Measurement` API here. 
So we can basically use [UnitLength](https://developer.apple.com/documentation/foundation/unitlength). This also will make our job easy so we can avoid having to do the conversion by hand.